packages/grid/src/BaseCell.tsx

```diff
@@ -12,6 +12,49 @@ export function getCellId<T>(rowKey: string, column: GridColumnModel<T>) {
   return `R${rowKey}C${column.info.props.id}`;
 }
 
+const icons = {
+  warning: (
+    <svg
+      aria-hidden
+      // width="1em"
```

Comment:
The attributes for the icons seem to be commented out randomly. Was this to fix scaling issues?

Reply:
It was yeah. I'll remove the commented out parts